SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -574,9 +545,9 @@ void UGlobalStateManager::BeginDestroy()
 #endif
 }
 
-bool UGlobalStateManager::HasAuthority()
+bool UGlobalStateManager::HasAuthority() const
 {
```

Comment:
Wondering if we should just do away with this method and force callers to specify the component they want to know the authority state of.

Reply:
I think there's only 1 case of this call outside the GSM